src/dataExplorer/components/SqlViewOptions.tsx

```diff
@@ -126,6 +127,19 @@ export const SqlViewOptions: FC<SqlViewOptionsT> = ({
               multiSelect={false}
               disabled={!selectedViewOptions?.smoothing?.applied}
             />
+            <RangeSlider
+              value={selectedViewOptions?.smoothing?.percentageRetained}
+              min={5}
+              max={100}
+              step={5}
+              onChange={event =>
+                selectViewOptions({
+                  smoothing: {percentageRetained: parseInt(event.target.value)},
+                })
+              }
+              labelPrefix="retained "
+              labelSuffix="%"
```

Comment:
Labels were added **after** the vids were made.
<img width="247" alt="Screen Shot 2023-01-24 at 11 42 46 AM" src="https://user-images.githubusercontent.com/10232835/214393117-4a199e1d-e017-4e6d-b80c-5f7f9ea72eb0.png">

Reply:
If this behaves like any of the other sliders in graph options, I think you might want to consider setting the value locally and then calling `selectViewOptions` after some amount of time, similarly to how it's done [here](https://github.com/influxdata/ui/pull/6284/files#diff-6bf175e958e1b62be5e473e5061ea15d322054a8a0e7e6ab18610b85c9cc6c48R163). I found that updating remote state that the graph depends on causes the experience to be incredibly janky.

Reply:
Good call. Definitely improves it.